src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -61,6 +61,19 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceBefore&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int c = a ++; // violation: '++' is preceded with whitespace
+ * int c = a++; // OK
+ * int foo
+ * ; // violation: allowLineBreaks is set false by default
```

Comment:
`allowLineBreaks is set false by default` -> line breaks are not allowed by default